File: dataset/output/OPENAI_gpt-4o/Task9_OPENAI_gpt-4o.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        List<Integer> list = Arrays.asList(0, -1, -5); // Example input list
        System.out.println(oddOrEven(list));
    }
    
    public static String oddOrEven(List<Integer> list) {
        int sum = list.stream().mapToInt(Integer::intValue).sum();
        return sum % 2 == 0 ? "even" : "odd";
    }
}

File: dataset/output/OPENAI_gpt-4o/Task9_OPENAI_gpt-4o.c
#include <stdio.h>

const char* oddOrEven(int arr[], int size) {
    int sum = 0;
    for(int i = 0; i < size; i++) {
        sum += arr[i];
    }
    return (sum % 2 == 0) ? "even" : "odd";
}

int main() {
    int arr[] = {0, -1, -5}; // Example input array
    int size = sizeof(arr) / sizeof(arr[0]);
    printf("%s
", oddOrEven(arr, size));
    return 0;
}